### DISCUSSION TITLE
Unable to obtain educational benefits

### DISCUSSION THREAD

**Discussion by ytbryan** [community/community#75693] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hey there, I tried to move an educational email from another account to mine but I couldn't get the education benefits. Can you please help me check my profile and case, please? 

**Update: To clarify, I'm a teacher and I have already verified my educational proof in my other account. However, moving my email to the current one does not port the educational benefits.**

**Comment by ishan-nahid** [comment]
Did you fill up the form they provide for applying as student?? 
Click on the Link: [https://education.github.com/pack](https://education.github.com/pack) 
Sign Up and give your information including the need of educational benefit in your projects. 
There is a trick in time of giving student prove. While giving ID card picture. take a selfi with id card. they value it more.

**Reply by ytbryan** [reply]
Thanks for the reply, it does not help. I have updated my case
